packages/autorest.gotest/test/integrationtest/output/appplatform/armappplatform/apps_client_example_test.go

```diff
@@ -32,11 +33,41 @@ func ExampleAppsClient_Get() {
 	if err != nil {
 		log.Fatalf("failed to finish the request: %v", err)
 	}
-	// TODO: use response item
+	// TODO: use response
 	_ = res
+	// If HTTP returns as the '200' response defined in example definition, response struct should like:
```

Comment:
should we also emphasize it here that some values below are fake and may or may not be in its proper format?
For instance, the `principalId`

Reply:
@xboxeer Could you help to refine the sentence here?

Reply:
If the HTTP response code is 200 as defined in example definition, your response structure would look like this. Please pay attention that all the values in the output are fake values for just demo purposes,